app/containers/message/Audio.js

```diff
@@ -175,7 +176,10 @@ class Audio extends React.Component {
 				>
 					<Video
 						ref={this.setRef}
-						source={{ uri }}
+						source={{
+							uri,
+							headers: RocketChatSettings.customHeaders
```

Comment:
Hey @regalstreak, how you can see [here](https://github.com/react-native-community/react-native-video#headers), headers on iOS doesn't work without a patch, can you try to change this component to use `Audio` from `Expo`?
https://docs.expo.io/versions/latest/sdk/audio/
The other solution (that I think is most difficult), is change the `RCTVideo.m` to use an allowed approach to headers.

Reply:
Yes, I'm aware of that as I had read the documentation. Skeptical about `RCTVideo.m` as we'd have to edit the library. Also, that is not there in the public API. Will look into the expo audio player for now. My suggestion would be to have minimum changes as possible here, as the audio component is going to be revamped during the GSoC any way.
@djorkaeffalexandre

Reply:
Expo-av will be done in a separate PR